src/main/default/cspTrustedSites/googleapis.cspTrustedSite-meta.xml

```diff
@@ -3,4 +3,10 @@
     <context>All</context>
     <endpointUrl>https://chart.googleapis.com</endpointUrl>
     <isActive>true</isActive>
+    <isApplicableToConnectSrc>true</isApplicableToConnectSrc>
```

Comment:
This is the only directive that needs to be set to true

Reply:
Right, a single directive will work as well.